web/public/locales/sv.json

```diff
@@ -1518,6 +1588,13 @@
     "UM": {
       "zoneName": "Förenta staternas mindre öar i Oceanien och Västindien"
     },
+    "US": {
+      "zoneName": "Kontinentala USA"
+    },
+    "US-AK": {
+      "countryName": "USA",
+      "zoneName": "Alaska"
+    },
```

Comment:
Just note that when the translations are identical they don't need to be added. It's not a problem though.